docs/spec/spec.md

```diff
@@ -600,7 +600,7 @@ HelloWorldClient securedEP = check new("https://localhost:9090",
 );
 ```
 
-#### 5.1.1.6 Client - Bearer Token Auth
+#### 5.1.1.6 Client - bearer token auth
```

Comment:
```suggestion
#### 5.1.1.6 Client - bearer token auth
```
Check the industrial terms.

Reply:
https://datatracker.ietf.org/doc/html/rfc6750